webpack/assets/javascripts/react_app/redux/actions/hosts/storage/vmware.js

```diff
@@ -60,30 +56,23 @@ export const changeCluster = newCluster => (dispatch, getState) => {
   dispatch(fetchStoragePods(config.storagePodsUrl, newCluster));
 };
 
-const fetchStorages = (url, cluster = null, actions = {}) => dispatch => {
+const fetchStorages = (url, cluster = null, key = '') => {
   if (cluster) {
-    ajaxRequestAction({
-      dispatch,
-      ...actions,
+    return {
+      type: API_OPERATIONS.GET,
+      key,
       url,
-      item: { params: { cluster_id: cluster } },
-    });
+      payload: { params: { cluster_id: cluster } },
+    };
   }
+  return {};
```

Comment:
This doesn't work, we are emiting empty object as an action here.
It's rather bad design on the first place, proposed fix: #7088

Reply:
Changed this to being dispatched but other than that the fix is outside of this PRs scope

Reply:
I agree, I just want it to work, I will fix it in a follow up PR :)